app/server-control/directives/kvm-console.js

```diff
@@ -30,11 +30,18 @@ window.angular && (function(angular) {
                 return false;
               };
 
+              var KVMconn = false;
               function connected(e) {
                 $log.debug('RFB Connected');
+                KVMconn = true;
               }
               function disconnected(e) {
                 $log.debug('RFB disconnected');
+                if (KVMconn === false) {
```

Comment:
Typo. It should be 2 equal signs, not 3: "if (KVMconn == false) {"

Reply:
I will revise it, thank you for your suggestion.

Reply:
I think it should be 3 
https://stackoverflow.com/questions/15393935/boolean-in-an-if-statement/15394481
https://codeburst.io/javascript-double-equals-vs-triple-equals-61d4ce5a121a

"Unless you’re very familiar with JavaScript, loose equality can lead to more headaches than it’s worth."

Reply:
I think in this case it should not matter but loose equality can lead to bugs, we had a gui code update bug around loose equality.